index.bs

```diff
@@ -1017,8 +1017,8 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 			<code>{ name: "clipboard-read", allowWithoutGesture: true }</code>
 			permission.
 
-		1. Let |hasGesture| be true if the current script is not running as a result of
```

Comment:
Why does `hasGesture` become true when the script runs *without* user gesture, though?

Reply:
I think that was a typo...